source/blog/articles/2019-07-onward-and-upward.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+
+title: "Onward and Upward!"
+date: "2019-07-02"
+author: "Nelson Shirk"
+image: "blog/2019/image-name.jpg"
+image_lg: "blog/2019/image-name.jpg"
+caption: "CAPTION TEXT HERE"
+
+---
+
+Wednesday was literally "breathtaking"! We started the day with a 3-hour train ride to the start of the trail that leads up to Mt/ Parashka. At the train station leaving Lviv, Kade befriended a German shepherd dog which then followed him onto the train!
+
+READMORE
+
+All went quite well. The dog managed to board without a ticket (which by the looks of him he couldn't afford one anyway) until one of the attendants came along and wasn't at all enthusiastic about the idea of having a stray dog on board. So it seemed that the dog must have got booted off at the next station as by the time we arrived at Parashka he was nowhere to be seen. 
+
+But soon after leaving the train and starting the hike another stray dog took a liking to us and joined in on the hike! The dog followed us all the way to the top (which was an exhausting 5.75 miles and a height of 2731 feet). The dog didn't seem the least bit phased by the climb and I must say strapping my pack to the dogs back was a thought that crossed my mind.
+
+Each of us had a pack weighing from 20 to 30 some pounds so the hike was no small challenge!
```

Comment:
Not sure why there is a "some" after "30". Should we move it before "30" or remove it?

Reply:
It's a colloquial way of saying 30+, but I agree it would be better removed.